stock_intercompany/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,4 @@
+* Go to Inventory > Settings
+* Under 'Operations' set
+
+  * Intercompany operation type
```

Comment:
add information about the creation of the picking type (which type of picking type has to be used, how to create it), on which company you need to set it.